site/public/js/notebook_builder/widgets/widget.js

```diff
@@ -1,6 +1,6 @@
 /* exported NUM_MARKDOWN, Widget */
 
-const NUM_MARKDOWN = 0;
+var NUM_MARKDOWN = 0;
```

Comment:
This fails our js-lint check, could you explain why we need to change it from const to var?

Reply:
The markdown-widget.js is changing the value on NUM_MARKDOWN each time a markdown is created, with a const variable I believe that you can't change that value

I don't know if the export at the top changes something with the variable

Reply:
I updated the variable from var to let to pass the js-lint check.